src/utils/gaa.js

```diff
@@ -292,9 +292,13 @@ export let GoogleUserDef;
 /**
  * Returns true if the query string contains fresh Google Article Access (GAA) params.
  * @param {string} queryString
+ * @param {boolean} allowAllAccessTypes
  * @return {boolean}
  */
-export function queryStringHasFreshGaaParams(queryString) {
+export function queryStringHasFreshGaaParams(
+  queryString,
+  allowAllAccessTypes = true
```

Comment:
Could we make this default to false now?

Reply:
Yep!